account/accounts.go

```diff
@@ -365,6 +370,53 @@ func (m *Manager) Accounts() ([]gethcommon.Address, error) {
 	return filtered, nil
 }
 
+// NewOnboarding starts the onboarding process generating accountsCount accounts and returns a slice of OnboardingAccount.
+func (m *Manager) NewOnboarding(accountsCount, mnemonicPhraseLength int) ([]*OnboardingAccount, error) {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+
+	onboarding, err := NewOnboarding(accountsCount, mnemonicPhraseLength)
+	if err != nil {
+		return nil, err
+	}
+
+	m.onboarding = onboarding
+
+	return m.onboarding.Accounts(), nil
+}
+
+// ResetOnboarding reset the current onboarding struct setting it to nil and deleting the accounts from memory.
+func (m *Manager) ResetOnboarding() {
```

Comment:
```suggestion
func (m *Manager) RemoveOnboarding() {
```

because reset suggest that this object is still available just its state got reset.